k8s/core/configmap/configmap_lock_v2.go

```diff
@@ -66,6 +66,7 @@ func (c *configMap) LockWithKey(owner, key string) error {
 
 	configMapLog(fn, c.name, owner, key, nil).Debugf("Starting lock refresh")
 	go c.refreshLock(owner, key)
+	lock.refreshing = true
```

Comment:
This is an indication of the routine's existence. Inside of refreshLock would be the right place for this assignment.

Reply:
Makes sense. We are not be able to check this value gets updated in the test anymore because the check may happen before the refreshing value gets assigned.